.nvmrc

```diff
@@ -0,0 +1 @@
+v18.18.0
```

Comment:
use 20?

Reply:
Whoops, committed from the wrong terminal!